provisioning/transport/amqp/tests/ProvisioningErrorDetailsAmqpTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Encoding;
+using Microsoft.Azure.Amqp.Framing;
+using Microsoft.Azure.Devices.Provisioning.Client.Transport;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Provisioning.Transport.Amqp.UnitTests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class ProvisioningErrorDetailsAmqpTests
+    {
+        private static TimeSpan defaultInterval = TimeSpan.FromSeconds(2);
+
+        [TestMethod]
+        public void getRetryFromRejectedSuccess()
+        {
+            int expectedSeconds = 32;
+            Rejected rejected = new Rejected();
+            rejected.Error = new Error();
+            rejected.Error.Info = new Fields();
+            rejected.Error.Info.Add(new Azure.Amqp.Encoding.AmqpSymbol("Retry-After"), expectedSeconds);
```

Comment:
missing `using` statement instead of the fully qualified name here.

Reply:
Also, names begin with caps.

Reply:
sure thing